src/Register.c

```diff
@@ -175,7 +175,7 @@ ZydisRegister ZydisRegisterGetLargestEnclosing(ZydisMachineMode mode, ZydisRegis
         [ZYDIS_REGCLASS_FLAGS] = { ZYDIS_REGISTER_FLAGS, ZYDIS_REGISTER_EFLAGS, ZYDIS_REGISTER_RFLAGS },
         [ZYDIS_REGCLASS_IP   ] = { ZYDIS_REGISTER_IP   , ZYDIS_REGISTER_EIP   , ZYDIS_REGISTER_RIP    },
     };
-    ZYAN_ASSERT(reg_class < ZYAN_ARRAY_LENGTH(STATIC_MAPPING));
+    ZYAN_ASSERT((ZyanU64)reg_class < ZYAN_ARRAY_LENGTH(STATIC_MAPPING));
```

Comment:
Shouldn't this be a cast to `ZyanUSize` instead? Otherwise I imagine that it'd still generate a warning in 32 bit builds.

Reply:
Thanks. 
It probably should be, because ZyanUSize (which ZYAN_ARRAY_LENGHT returns) and ZyanU64 can differ in size.

Reply:
Ok, cool -- can you change the PR accordingly, please?

Reply:
Yes. Changed it. Thank you for your time :)